app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -77,145 +96,125 @@ const PageRequestReview = ({
     },
   );
 
-  return (
-    <AppLayoutWithExperiment
-      title="Review &amp; Launch"
-      testId="PageRequestReview"
-      {...{ polling }}
-      redirect={({ status, review }) => {
-        if (review && status.draft && !review.ready) {
-          // If the experiment is not ready to be reviewed, let's send them to
-          // the first page we know needs fixing up, with field errors displayed
-          return `edit/${review.invalidPages[0] || "overview"}?show-errors`;
-        }
-
-        if (status.launched) {
-          // Return to the experiment root/summary page
-          return "";
-        }
-      }}
-    >
-      {({ experiment, refetch }) => {
-        currentExperiment.current = experiment;
-        refetchReview.current = refetch;
-        const status = getStatus(experiment);
-        const {
-          publishStatus,
-          canReview,
-          reviewRequest: reviewRequestEvent,
-          rejection: rejectionEvent,
-          timeout: timeoutEvent,
-        } = experiment;
+  const {
+    publishStatus,
+    canReview,
+    reviewRequest: reviewRequestEvent,
+    rejection: rejectionEvent,
+    timeout: timeoutEvent,
+  } = experiment;
 
-        return (
-          <>
-            {submitError && (
-              <Alert data-testid="submit-error" variant="warning">
-                {submitError}
-              </Alert>
-            )}
+  return (
+    <>
+      {submitError && (
+        <Alert data-testid="submit-error" variant="warning">
+          {submitError}
+        </Alert>
+      )}
 
-            {(status.draft || status.preview) && (
-              <ChangeApprovalOperations
+      {(status.draft || status.preview) && invalidPages.length > 0 ? (
+        <Alert variant="warning">
+          Before this experiment can be reviewed or launched, all required
+          fields must be completed. Fields on <InvalidPagesList /> are missing
```

Comment:
Definitely not blocking, but I feel like the phrasing here could be better? "Fields on **the** [pages] **page(s)** are missing details."